rs_client.py

```diff
@@ -1019,6 +1019,7 @@ def _rs_struct(typeobj):
 
         _r('pub type %s%s = base::StructPtr<%s%s>;', typeobj.rs_type, lifetime1,
                 lifetime2, typeobj.ffi_type)
+        typeobj.struct_ptr = True
 
```

Comment:
A few remarks for this:
 1. Please prefix added attributes with `rs_` (i.e. `rs_struct_ptr`).
 2. Those assignments must be done in `rs_type_setup` (not all types are going through `_rs_struct`) and all type objects should have either `True` or `False` assigned (here only `True` or attribute undefined). This will avoid unpleasant test like `hasattr(field.type, 'struct_ptr')`.
 3. That one is actually not necessary :smile:. Decision between StructPtr or POD is made by attribute `rs_is_pod`. I.e. `typeobj.struct_ptr == not typeobj.rs_is_pod`.